examples/cfrl_mnist.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<p>Typo: experince --&gt; experience</p><p> </p><p>In general, I recommend installing a spell-check extension to the Jupyter notebook, e.g. https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/spellchecker/README.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>